api/docs/release.dox

```diff
@@ -219,6 +219,7 @@ Further non-compatibility-affecting changes include:
    either XMM, YMM or ZMM, excluding any MMX register checks.
  - Added DR_NUM_SIMD_VECTOR_REGS as an alias to MCXT_NUM_SIMD_SLOTS in order
    to get the static number of supported SIMD vectors.
+ - Added dr_raw_read_tls_slot() and dr_raw_write_tls_slot().
```

Comment:
If this interface is going to be added, the sample clients showing how to use our API should be changed to use these functions: instrace_simple, memtrace_simple.  That also makes sure this interface is what a client would want.

Additionally, there are uses in drx_buf, drcachesim's tracer, Dr. Memory, and perhaps other places that should be examined and possibly updated.  Do any of those need to touch a different thread's slots?  Dr. Memory certainly does that for some of its TLS; I would guess it does for raw TLS slots but have not checked the code.